lib/valkyrie.rb

```diff
@@ -86,13 +86,37 @@ def storage_adapter
       Valkyrie::StorageAdapter.find(super.to_sym)
     end
 
+    # @api public
+    # @since 2.0.0.rc1
```

Comment:
It's probably safer to omit this. This is backwards compatible, it'll probably be in 1.5

Reply:
I can remove the `@since`.